.github/workflows/deploy-docs.yaml

```
@@ -32,7 +37,7 @@ jobs:
 
       - name: Derive files from sources
         run: |
-          make squeaky-clean all
+          make  all
```

I suggest removing this extra space.

```suggestion
          make all
```